examples/large_models/vllm/mistral/model-config.yaml

```diff
@@ -1,11 +1,10 @@
 # TorchServe frontend parameters
 minWorkers: 1
 maxWorkers: 1
-batchSize: 16
 maxBatchDelay: 100
 responseTimeout: 1200
 deviceType: "gpu"
-continuousBatching: true
+asyncCommunication: true
```

Comment:
What happens if a user specifies batchSize ? i.e Does this take care of ignoring dynamic batching in frontend?

Reply:
Choosing batch_size != 1 will currently trigger an assertion in the handler and there is not point in supporting it as it only delays processing of the request.